### DISCUSSION TITLE
Nextjs 13 (13.0.6) Fetch Other Route at First Time when "standalone" mode

### DISCUSSION THREAD

**Discussion by barayuda** [vercel/next.js#43959] [Help] [answered]
Dear Nextjs team,
Thank you for amazing Nextjs 13
And support "standalone" mode as well when build production in version 13.0.6

I have minor issue, when succeeded build and access the production in my self-hosting server (GCP with docker build), I got at first time that the website fetch "all route" before the homepage open, like picture below

![image](https://user-images.githubusercontent.com/520463/207055211-a1b4c410-8b11-43f4-b98c-43a1f813600a.jpeg)

Is it expected?

**Comment by soylemezali42** [comment, marked as answer]
In production, Link components are using prefetch props to optimize the pages. It is expected and normal. You can disable this behaviour if it is not suitable for your usecase.

**Reply by barayuda** [reply]
thanks for replay @soylemezali42 , I found solution here https://github.com/vercel/next.js/discussions/24009
